code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1261,11 +1261,13 @@
 		var/bodypart_damages = list()
 		//Loop through all external organs and save the damage states for brute and burn
 		for(var/obj/item/organ/external/E in bodyparts)
-			if(E.brute_dam == 0 && E.burn_dam == 0 && E.internal_bleeding == FALSE) //If there's no damage we don't bother remembering it.
+			if(E.brute_dam == 0 && E.burn_dam == 0 && !(E.status & ORGAN_INT_BLEEDING)) //If there's no damage we don't bother remembering it.
 				continue
 			var/brute = E.brute_dam
 			var/burn = E.burn_dam
-			var/IB = E.internal_bleeding
+			var/IB
+			if(E.status & ORGAN_INT_BLEEDING)
+				IB = TRUE
```

Comment:
```suggestion
			var/IB = (E.status & ORGAN_INT_BLEEDING)
```

Reply:
This would give it the value `ORGAN_INT_BLEEDING`. Which is fine if it's just used for an `if` statement later which most likely will be the solution

Reply:
Huh did not know that, but yeah it should be fine in this case.

Reply:
It also looks like this code hasn't been updated from when broken  bones and such were added. so it only carries IB over on transforms. should i fix that in this PR or move it to a different one?